### ISSUE TITLE
gz-utils: Installation

### ISSUE THREAD

**Issue by azeey** [gazebosim/gazebo_test_cases#436]
## Setup
- Os: Ubuntu Jammy
- BuildType: Source

## Links
- [Live](https://gazebosim.org/api/utils/2/install.html)
- [Source](https://github.com/gazebosim/gz-utils/tree/gz-utils2/tutorials/installation.md)

## Checks
- [ ] **Instructions work**
- [ ] **Images (if there are any) match the result**

---

*You can find the code used to generate this test case [here](https://github.com/audrow/yatm)*

**Comment by Kartik9250** [comment]
I would like to work on this

**Comment by Kartik9250** [comment]
As i mention in the [gz-utils: reame](https://github.com/gazebosim/gazebo_test_cases/issues/438#issuecomment-1708078354)
- the code snippets are not loading properly on the website.
- the second code snippet gives the following output:
   ![image](https://github.com/gazebosim/gazebo_test_cases/assets/58456392/004cfabc-820d-4c56-9c51-a75bf54007a3)

**Comment by Voldivh** [comment]
Thanks @Kartik9250 I'll be taking a look at it.

**Comment by Kartik9250** [comment]
Okay on a closer look. I found:
 - The `live` and the `source` differs from one another.
 - The `step 2` under the `source installation` section has a "`" missing after `gz-utils<#>`, and one before `replace<#>`
 - The `step 3` is probably rendering `\n` as a new line which is after "tr `"
 
 I can resolve these but the `source` and `live` differs. I am not sure which one is the live branch as in the markdown of the branches that I checked they seemed fine.
